biocrnpyler/chemical_reaction_network.py

```diff
@@ -271,7 +271,7 @@ def replace_species(self, species: Species, new_species: Species):
 
         return ChemicalReactionNetwork(new_species_list, new_reaction_list)
 
-    def generate_sbml_model(self, stochastic_model=False, show_warnings = False, **keywords):
+    def generate_sbml_model(self, stochastic_model=False, show_warnings = False, check_validity = True,**keywords):
```

Comment:
sorry for asking these questions, but why you need this check_validity flag? Is it for speed improvement again?

Reply:
yes correct. "Check validity" is pretty slow so my work around is just not do it :). I am not 100% sure why it is needed but @WilliamIX can probably tell you more

Reply:
check_validity calls an SBML function (from lib_sbml) which ensures that the final model is valid. By default this should definitely be on - it is particular important in ensuring that general propensity strings "make sense". For specific test cases, this could be False if the test is not testing SBML writing.

Reply:
Okay, I got it now.